css/stylesheet.css

```diff
@@ -28,6 +28,7 @@ main {
 ul {
   border: 1px solid #555;
   text-align: center;
+  hei
 }
 
 ul > li {
```

Comment:
Since you're using this for the nav section, so you can use "nav li" to be more specific and semantic.